apps/server/src/modules/common-cartridge/controller/dto/common-cartridge-import-body.params.ts

```diff
@@ -0,0 +1,11 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CommonCartridgeImportBodyParams {
+	@ApiProperty({
+		type: String,
+		format: 'binary',
+		required: true,

```

Comment:
This can be removed. Default value of required in ApiProperty is true.

Reply:
Only the required or the entire ApiProperty decorator?

Reply:
The required property only

https://docs.nestjs.com/openapi/types-and-parameters#:~:text=Instead%20of%20explicitly%20typing%20the%20%40ApiProperty(%7B%20required%3A%20false%20%7D)%20you%20can%20use%20the%20%40ApiPropertyOptional()%20short%2Dhand%20decorator.

Take a quick look at this link, it will confirm Firas' statement

Reply:
Done ✅